src/app/components/form/input/state/FormInputStateWarning.vue

```diff
@@ -1,6 +1,5 @@
 <template>
   <FormInputState class="text-yellow-600">
```

Comment:
I think we can use the equivalent `--semantic-warning-strong` (and `--semantic-success-strong` in the other component) colors here too